### ISSUE TITLE
Error: start must be between 0 and 255; given Infinity

### ISSUE THREAD

**Issue by DTiunov** [maptiler/tileserver-gl#1352]
Hello.
I updated my GL tileserver to version 4.12.0. And now I always get the error:
```
mlgl: {
  class: 'Style',
  severity: 'ERROR',
  text: 'Failed to load glyph range 39168-39423 for font stack Roboto Condensed Italic,Roboto Italic: start must be between 0 and 255; given Infinity'
}
mlgl: {
  class: 'Style',
  severity: 'ERROR',
  text: 'Failed to load glyph range 38656-38911 for font stack Roboto Condensed Italic,Roboto Italic: start must be between 0 and 255; given Infinity'
}
```
when I execute the request  `/styles/peaksummer-jpg/static/121.272039,24.388722,12/700x160.png`
This always happens with the Roboto Condensed Italic font. I tried to convert the font, but there was no result.

This looks like a bug to me. Perhaps I'm wrong. I'm glad to get any help.

**Comment by acalcutt** [comment]
Looks like i created a duplicate of this in https://github.com/maptiler/tileserver-gl/issues/1367

It seems to be a combined font issue in the @jsse/pbfont we switched to to try and work around @mapbox/glyph-pbf-composite not working past node 20.14.0.

**Comment by akx** [comment]
Apparently fixed via https://github.com/maptiler/tileserver-gl/pull/1369?

**Comment by acalcutt** [comment]
This should be fixed in [v4.13.0](https://github.com/maptiler/tileserver-gl/releases/tag/v4.13.0)
